catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
We should use [Unofficial Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/), or at least use `set -euo pipefail`

Reply:
#1 - Why'd you comment the same thing twice?
#2 - when i try adding that to the script, it immediatly returns a syntax error, saying that -euo is an invalid option. and yes, i'm running it through bash

Reply:
1. GitHub moment not showing the first comment after refreshing the new changes.
2. How are you adding it?
```bash
#!/usr/bin/env bash
set -euo pipefail
IFS=$'\n\t'
```
works on my machine

Reply:
just like how you did it

Reply:
@TheBearodactyl Are you sure you're doing `pipefail` and not `pipefall`?

Reply:
yes